build/windows/10.1.0/sitecore-roles/sitecore-custom-solr-init/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# escape=`
+ARG BASE_IMAGE
+ARG MODULE_ASSETS
+
+FROM ${MODULE_ASSETS} as module_assets
+
+FROM $BASE_IMAGE as final
+
+ARG MODULE_NAME
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+COPY --from=module_assets C:\module\solr\cores-$MODULE_NAME.json C:\data\cores-$MODULE_NAME.json
```

Comment:
Given that there is a breaking change for 10.1 on Managed Schemas not populated. Do you think this might be a good spot to actually help automatically populate the Managed Schema ?

Reply:
Hey @vtml  - this is definitely a nice-to-have but probably not a quick fix. We'll merge as-is and look to improve this in the future.